src/components/KSlideout/KSlideout.vue

```diff
@@ -3,7 +3,7 @@
     <transition name="fade">
       <div
         v-if="isVisible"
-        class="panel-background"
+        :class="{ 'panel-background': overlayEnabled }"
         @click="(event: any) => handleClose(event, true)"
```

Comment:
question: does this actually work like we want? I don't think it's enough to conditionally apply a class to the overlay div because we are still catching all click events and using it to toggle the visibility of the slideout which is not what we want. If overlay is disabled, there should be no overlay div and clicks on background content should actually be applied to the background content.